server/client.go

```diff
@@ -932,6 +1045,11 @@ func (c *client) processMsg(msg []byte) {
 	}
 }
 
+func (c *client) pubPermissionViolation(subject []byte) {
+	c.sendErr(fmt.Sprintf("Permissions Violation for Publish to %q", subject))
+	c.Errorf("Publish Violation - User %q, Subject %q", c.opts.Username, subject)
```

Comment:
I see your point about the connect, but in this case a simple user error (wrong subject) with typical API usage could end up being really harmful to the server.  How would you feel about making it a debug statement for now, then promoting it to an error when thresholds are implemented?